src/main/java/edu/cornell/kfs/module/purap/CUPurapKeyConstants.java

```diff
@@ -28,7 +28,7 @@ public class CUPurapKeyConstants {
 
     public static final String ATTACHMENT_QUESTION_CONFIRM_CHANGE = "attachment.message.confirm.change";
     
-    public static final String MESSAGE_IWNT_CONFIRM_CREATE_REQ = "message.createReq.confirmationMessage";
+    public static final String MESSAGE_IWNT_CONFIRM_CREATE_REQ_OR_DV = "message.createReqOrDV.confirmationMessage";

```

Comment:
Should the "V" in "message.createReqOrDV.confirmationMessage" be lower case for consistent camel casing?

Reply:
Good question. I find it easier to read as is but I can change it.